config/routes.rb

```diff
@@ -15,12 +15,11 @@
     /conversations
     /lists/(*any)
     /notifications
-    /favourites
-    /bookmarks
```

Comment:
This would abruptly break it for users that have bookmarked one of these pages or have a saved browser session.